portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,16 +56,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     if (sort) {
       Arrays.sort(words);
     }
-
-      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  //storing the string in database
+      
     StringList.add(Arrays.toString(words));
 
+        //Creating a new entity to place in the database
     Entity taskEntity = new Entity("Task");
-   
-    taskEntity.setProperty("Comments", StringList);
 
+   LocalDateTime localdate = LocalDateTime.now();
+
+    String dateandtime = " " + localdate;
+    taskEntity.setProperty("Comments", StringList);
```

Comment:
In the future, you should be more consistent with capitalization, but this is OK for now.

Reply:
Ok thanks I actually didnt even us that part, but it is good to know